src/components/templates/Jazzbar/components/JazzBarRoom/LocalParticipant.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react";
```

Comment:
This file seems to duplicate our generic component `src/components/organisms/Room/LocalParticipant.tsx` (which is already duplicating `src/components/templates/Playa/Video/LocalParticipant.tsx`, but that is tech debt to be deleted soon).

What's the reasoning/need for duplicating it? As best I can tell from a quick visual skim, they seem to be exactly the same?

Reply:
deleted in https://github.com/sparkletown/sparkle/pull/1135/commits/0aa3e561fb259e930158b63b5f6757b11be37d7d